python/pylibcudf/tests/common/utils.py

```diff
@@ -183,7 +183,23 @@ def _is_supported_for_pc_is_nan(arr_type):
 
             np.testing.assert_array_almost_equal(lh_arr, rh_arr)
     else:
-        assert lhs.equals(rhs)
+        if not lhs.equals(rhs):
```

Comment:
Just FYI on line 176 there's another `assert lhs_nans.equals(rhs_nans)` that could use the same treatment.